src/librbd/ImageCtx.cc

```diff
@@ -237,34 +237,34 @@ namespace librbd {
   void ImageCtx::perf_start(string name) {
     PerfCountersBuilder plb(cct, name, l_librbd_first, l_librbd_last);
 
-    plb.add_u64_counter(l_librbd_rd, "rd");
-    plb.add_u64_counter(l_librbd_rd_bytes, "rd_bytes");
-    plb.add_time_avg(l_librbd_rd_latency, "rd_latency");
-    plb.add_u64_counter(l_librbd_wr, "wr");
-    plb.add_u64_counter(l_librbd_wr_bytes, "wr_bytes");
-    plb.add_time_avg(l_librbd_wr_latency, "wr_latency");
-    plb.add_u64_counter(l_librbd_discard, "discard");
-    plb.add_u64_counter(l_librbd_discard_bytes, "discard_bytes");
-    plb.add_time_avg(l_librbd_discard_latency, "discard_latency");
-    plb.add_u64_counter(l_librbd_flush, "flush");
-    plb.add_u64_counter(l_librbd_aio_rd, "aio_rd");
-    plb.add_u64_counter(l_librbd_aio_rd_bytes, "aio_rd_bytes");
-    plb.add_time_avg(l_librbd_aio_rd_latency, "aio_rd_latency");
-    plb.add_u64_counter(l_librbd_aio_wr, "aio_wr");
-    plb.add_u64_counter(l_librbd_aio_wr_bytes, "aio_wr_bytes");
-    plb.add_time_avg(l_librbd_aio_wr_latency, "aio_wr_latency");
-    plb.add_u64_counter(l_librbd_aio_discard, "aio_discard");
-    plb.add_u64_counter(l_librbd_aio_discard_bytes, "aio_discard_bytes");
-    plb.add_time_avg(l_librbd_aio_discard_latency, "aio_discard_latency");
-    plb.add_u64_counter(l_librbd_aio_flush, "aio_flush");
-    plb.add_time_avg(l_librbd_aio_flush_latency, "aio_flush_latency");
-    plb.add_u64_counter(l_librbd_snap_create, "snap_create");
-    plb.add_u64_counter(l_librbd_snap_remove, "snap_remove");
-    plb.add_u64_counter(l_librbd_snap_rollback, "snap_rollback");
-    plb.add_u64_counter(l_librbd_notify, "notify");
-    plb.add_u64_counter(l_librbd_resize, "resize");
-    plb.add_u64_counter(l_librbd_readahead, "readahead");
-    plb.add_u64_counter(l_librbd_readahead_bytes, "readahead_bytes");
+    plb.add_u64_counter(l_librbd_rd, "rd", "Iterate reads");
```

Comment:
these three librbd_rd counters are for read() as well as read_iterate(). I'd just leave iterate out of the description